app/controllers/webhook_deliveries_controller.rb

```diff
@@ -19,6 +19,32 @@ def index
     @error = e.message
   end
 
+  def show
+    # TODO: unstub
+    @webhook_delivery = {"id" => "210e2eb8-af39-495f-baba-ca589860ea07",
```

Comment:
Can unstub after this [Gravity Endpoint is released](https://github.com/artsy/gravity/pull/18225)